near-sdk/tests/test-contracts/store/src/lib.rs

```diff
@@ -0,0 +1,215 @@
+#![allow(deprecated)]
+
+use near_sdk::borsh::{BorshDeserialize, BorshSerialize};
+use near_sdk::serde::{Deserialize, Serialize};
+use near_sdk::sys::panic;
+use near_sdk::{near, near_bindgen, store, PanicOnDefault};
+use Collection::*;
+
+#[near(contract_state)]
+#[derive(PanicOnDefault)]
+// This contract is designed for testing all of the `store` collections.
+pub struct StoreContract {
+    pub iterable_set: store::IterableSet<u32>,
+    pub iterable_map: store::IterableMap<u32, u32>,
+    pub unordered_set: store::UnorderedSet<u32>,
+    pub unordered_map: store::UnorderedMap<u32, u32>,
+    pub tree_map: store::TreeMap<u32, u32>,
+    pub lookup_map: store::LookupMap<u32, u32>,
+    pub lookup_set: store::LookupSet<u32>,
+    pub vec: store::Vector<u32>,
```

Comment:
Let's use more complex type than u32 for the load testing, since it is rare to see collections of primitive types in contracts.